### PR TITLE
Added an example app with a supporting Redis container in a separate pod

### PR DESCRIPTION
**Pull request by hosainnet**
Sample Redis configuration

### CODE REVIEW

**Comment by hosainnet** [comment]
@timwebster9 @timja

**Review by timja — APPROVED** [review]
lgtm @timwebster9 ?

File: redis/app-redis-separate-pods.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+          value: "redis://${SERVICE_NAME}-redis:6379"
+        # These may be adjusted, but be mindful please
+        resources:
+          requests:
+            memory: "64Mi"
+            cpu: "200m"
+          limits:
+            memory: "256Mi"
+            cpu: "500m"
+
```
Comment (timwebster9) [inline redis/app-redis-separate-pods.yaml:45]: Just out of curiousity are these resource limits accurate for Redis or just copy/pasted?

File: redis/app-redis-separate-pods.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      - name: master
+        image: redis:5.0-alpine
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
+        ports:
+        - containerPort: 6379
+#
+# Redis binding
```
Comment (timwebster9) [inline redis/app-redis-separate-pods.yaml]: This should probably say 'Redis Service'

File: redis/app-redis-separate-pods.yaml (hunk 1 above)
Comment (hosainnet) [reply redis/app-redis-separate-pods.yaml:45]: These were largely taken from https://kubernetes.io/docs/tutorials/stateless-application/guestbook/#start-up-the-redis-master

Comment (hosainnet) [reply redis/app-redis-separate-pods.yaml]: 👍

File: redis/app-redis-separate-pods.yaml (hunk 2 above)
Comment (hosainnet) [reply redis/app-redis-separate-pods.yaml]: 👍

**Comment by hosainnet** [comment]
Hi, can we merge this please (or close it if AKS is no longer relevant)? Thanks :)

**Pull request merged**